skimage/data/__init__.py

```diff
@@ -592,6 +592,63 @@ def checkerboard():
     return _load("data/chessboard_GRAY.png")
 
 
+def cells3D():
+    """3D microscopy images of cells.
+
+    The data for this was provided by the Allen Institute for Cell Science.  It
```

Comment:
I think information like 598–609 should be in a `Notes` section.

Reply:
Understanding your data is critical to good analysis and visualization.

What is the difference between the "Notes" and the "Description".

If you feel strongly about it, do reorganize.

Reply:
We follow the [Numpy docstyle](https://numpydoc.readthedocs.io/en/latest/format.html). According to them:
> The sections of a function’s docstring are:
>  `1.` Short summary
>  A one-line summary that does not use variable names or the function name (...)
> (...)
> `13.` Notes
> An optional section that provides additional information about the code, possibly including a discussion (...)

Reply:
I've made some modifications, however, I will point to:

> Extended Summary
> 
> A few sentences giving an extended description. This section should be used to clarify functionality, not to discuss implementation detail or background theory, which should rather be explored in the Notes section below. You may refer to the parameters and the function name, but parameter descriptions still belong in the Parameters section.

Functionality here, for me, represents things like metadata, depicting voxel size, which are simply not communicated in a numpy array. I've kept those details.